src/Html/Builder/Select.php

```diff
@@ -626,8 +639,7 @@ public static function options($arr, $optKey = 'value', $optText = 'text', $sele
 					$attr = trim($attr);
 				}
 
-				$extra = ($id ? ' id="' . $id . '"' : '') . ($label ? ' label="' . $label . '"' : '') . ($attr ? ' ' . $attr : '') . $extra;
-
+				$extra = ($id ? ' id="' . $id . '"' : '') . ($label ? ' label="' . $label . '"' : '') . ($attr ? ' ' . $attr : '') . $dataAttributes;
```

Comment:
This looks like it dropped the ` . $extra` part (which usually has `disabled="disabled"`). I think that should still be in there? `) . $extra . $dataAttributes;`